src/index.html

```diff
@@ -11,7 +11,221 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    onsubmit="onSubmit()"
+    method="post"
+    >
+
+      <fieldset class="block">
+        <legend>Personal information</legend>
+
+        <div class="input">
+          <label for="surname">
+            Surname:
+            <input
+            type="text"
+            id="surname"
+            name="surname"
+            autocomplete="off"
+          >
```

Comment:
```suggestion
            <input
              type="text"
              id="surname"
              name="surname"
              autocomplete="off"
            >
```
у атрибутов один отступ(2 пробела), закрывающая скобка на одной линии с отркывающей